docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -368,6 +907,10 @@ The export data from source command first ensures that it exports a snapshot of
 Note that the CDC phase will start only after a snapshot of the entire interested table-set is completed.
 Additionally, the CDC phase is restartable. So, if yb-voyager terminates when data export is in progress, it resumes from its current state after the CDC phase is restarted.
 
+{{<note title="Important">}}
+yb-voyager creates a replication slot in the source database where disk space can be used up rapidly. To avoid this, execute the [Cutover to the target](#cutover-to-the-target) or [End Migration](#end-migration) steps to delete the replication slot. If you choose to skip the steps, then you must delete the replication slot manually to reduce disk usage.
```

Comment:
```suggestion
yb-voyager creates a replication slot in the source database where disk space can be used up rapidly. To avoid this, execute the [Cutover to the target](#cutover-to-the-target) or [End Migration](#end-migration) steps to delete the replication slot. If you choose to skip these steps, then you must delete the replication slot manually to reduce disk usage.
```